.goreleaser.yaml

```diff
@@ -13,7 +13,7 @@ builds:
       - amd64
       - arm64
     env:
-      - CGO_FLAGS="-O2 -D__BLST_PORTABLE__"
+      - CGO_CFLAGS="-O 2 -D __BLST_PORTABLE__"
```

Comment:
extra space?
```suggestion
      - CGO_CFLAGS="-O2 -D __BLST_PORTABLE__"
```

Reply:
you can also use `CGO_CFLAGS="-O -D __BLST_PORTABLE__"`. 
The `2` just adds further optimizations

Reply:
awesome thanks, yeah I'll try that